gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -546,6 +546,7 @@ gst_tensor_filter_properties_init (GstTensorFilterProperties * prop)
   prop->custom_properties = NULL;
   prop->accl_str = NULL;
   prop->hw_list = NULL;
+  prop->shared_tensor_filter_key = NULL;
```

Comment:
Or... do memsetzero to *prop and set non-zero values only for the future safety?

Reply:
updated